root/programs/C#/Frameworks/Infrastructure/Public/Util/GetPassword.cs

```diff
@@ -0,0 +1,121 @@
+﻿//**********************************************************************************
+//* Copyright (C) 2007,2016 Hitachi Solutions,Ltd.
```

Comment:
@yusukemaegawa 

For newly added classes, change from 2016 to 2017.
https://oshiete.goo.ne.jp/qa/5528706.html